src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst

```diff
@@ -0,0 +1,252 @@
+ 
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_WellboreMesh -->
+  :end-before: <!-- SPHINX_WellboreMeshEnd -->
+
+.. _meshThermoPoroElasticWellboreFig:
+.. figure:: mesh.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   An optimized mesh for the open wellbore.
+
+-----------------------------------------------------------
+Material properties
+-----------------------------------------------------------
+
+The bulk and shear drained elastic moduli of rock as well as its drained linear thermal expansion coefficient relating stress change to temperature change are defined within the ``Constitution`` tag as follows:
+ 
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_ThermoElasticProperties -->
+  :end-before: <!-- SPHINX_ThermoElasticPropertiesEnd -->
+
+Here the solid density is also defined but it is not used because the gravitational effect is ignored in this example. The porosity and the elastic bulk modulus :math:`K_{s}` of the solid skeleton are defined as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PoroElasticProperties -->
+  :end-before: <!-- SPHINX_PoroElasticPropertiesEnd -->
+
+The thermal conductivities and the volumetric heat capacities of rock are defined by following XML blocks:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_ThermalProperties -->
+  :end-before: <!-- SPHINX_ThermalPropertiesEnd -->
+
+and
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_HeatCapacityProperties -->
+  :end-before: <!-- SPHINX_HeatCapacityPropertiesEnd -->
+
+The permeability of rock is defined by:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PermeabilityProperties -->
+  :end-before: <!-- SPHINX_PermeabilityPropertiesEnd -->
+
+Fluid properties such as viscosity, thermal expansion coefficient, etc. are defined by the XML block below. A negligible volumetric heat capacity is defined for fluid to ignore the thermal convection effect such that only thermal transfer via the diffusion phenomenon is considered.
```

Comment:
The part about the volumetric heat capacity seems to come from the other tutorial, but I don't think you have a negligible heat capacity here based on your xml file, right?